README.md

```diff
@@ -18,7 +18,9 @@ If you have an existing `fly.toml` in your repo, this action will copy it with a
 | `path`     | Path to run the `flyctl` commands from. Useful if you have an existing `fly.toml` in a subdirectory.                                                                                                     |
 | `postgres` | Optional name of an existing Postgres cluster to `flyctl postgres attach` to.                                                                                                                            |
 | `update`   | Whether or not to update this Fly app when the PR is updated. Default `true`.                                                                                                                            |
-| `secrets`  | Secrets to be set on the app. Separate multiple secrets with a space                                                                                                                                     |
+| `secrets`  | Secrets to be set on the app at runtime. Separate multiple secrets with a space                                                                                                                                     |
+| `build_args` | Optional Docker --build-arg |
```

Comment:
if we add this option we don't need to pass `dockerfile` option explicitly anymore ?

Reply:
I guess the answer to your question is yes. I don't have enough context to give you a straight forward good quality answer tho.
my use case is as follow:

I use `build_args` to set, inside the image, as an environment variable:

-  the `commit sha`. I use this because I need to tell apart which sourcemap sent to Sentry correspond to which build
- a flag `autosleep`. My Remix server checks at runtime if that flag is there and if so it shuts itself down after a while in order to scale the fly machine down

`build_secrets` I use to set credentials needed by Sentry to upload the sourcemaps

and finally my dockerfile is the same for build and for production but the generated image is different because I make use of `FROM` more than once in the same dockerfile 

> FROM can appear multiple times within a single Dockerfile to create multiple images or use one build stage as a dependency for another. Simply make a note of the last image ID output by the commit before each new FROM instruction. Each FROM instruction clears any state created by previous instructions.

https://docs.docker.com/reference/dockerfile/#from

Reply:
Got it , thanks